app/src/main/java/org/fossasia/openevent/app/event/attendees/AttendeesFragment.java

```diff
@@ -178,6 +178,18 @@ public void onChanged() {
             }
         };
         stickyHeaderAdapter.registerAdapterDataObserver(adapterDataObserver);
+
+        recyclerView.addOnScrollListener(new RecyclerView.OnScrollListener() {
```

Comment:
This may get used in many upcoming fragments/activities. Please add a static method in ViewUtils with RecyclerView and FAB as argument